lib/new_relic/agent/transaction/tracing.rb

```diff
@@ -6,7 +6,7 @@ module NewRelic
   module Agent
     class Transaction
       module Tracing
-        attr_reader :current_segment
+        attr_reader :current_segment_by_thread
```

Comment:
I can't find where this reader is used

Reply:
It's used in the new methods in Transaction for current segment. Those could be updated to be @current_segment_by_thread though, and remove this reader since those are the only places using it.

Reply:
Oh, I see what happened. I typed in "current_segment_by_id " for my search for some reason. Keeping it is ok by me.

Reply:
Ahh okay that makes sense! Sounds good!